ods_ci/tests/Tests/400__ods_dashboard/413__ods_dashboard_user_mgmt.robot

```diff
@@ -49,28 +49,25 @@ Verify If Unauthorized User Can Not Change The Permission
 Verify Unauthorized User Is Not Able To Spawn Jupyter Notebook
     [Documentation]    Verify unauthorized User Is Not Able To Spawn Jupyter
     ...     Notebook , user should not see a spawner if the user is not in allowed Groups
+    ...     Note: this test creates users/groups via ODH Dashboard UI in
```

Comment:
it doesn't create users nor groups, it configures the user access of the dashboard

Reply:
ah, yeah - I'll reword that, good catch, thanks!